.gitignore

```diff
@@ -0,0 +1,2 @@
+test.md
+node_modules/.
```

Comment:
[OPTIONAL] You have done wonderful work so far but kindly know that ordinary single space matters in coding and it might break a whole code. In this case, the `.` at the back of node_modules is breaking the code and my system is unable to withstand it. My system hanging. The `.` has given another message. Kindly remove the `.` and you will see that the files would reduce from 6000 plus. We ignore node_modules because it has a large size.